cmd/loadtest/submitter/submitter.go

```diff
@@ -118,10 +132,19 @@ func (s *Submitter) submit(logger log.Logger) error {
 	)
 
 	for i := 0; i < s.opsPerTx; i++ {
-		ops = append(ops, build.Payment(
-			build.Destination{AddressOrSeed: s.destinationAddress},
-			build.NativeAmount{Amount: s.transferAmount}),
-		)
+		var payment build.PaymentBuilder
+
+		if native {
+			payment = build.Payment(
+				build.Destination{AddressOrSeed: s.destinationAddresses[destIndex].Address()},
+				build.NativeAmount{Amount: s.transferAmount})
+		} else {
+			payment = build.Payment(
+				build.Destination{AddressOrSeed: s.destinationAddresses[destIndex].Address()},
+				build.CreditAmount{"KIN", "GBSJ7KFU2NXACVHVN2VWQIXIV5FWH6A7OIDDTEUYTCJYGY3FJMYIDTU7", s.transferAmount})
```

Comment:
this should be configurable or at least hard coded

Reply:
also define `PaymentMutation` with either `NativeAmount` or `CreditAmount` in the if statement and outside of it assign it to the `Payment()` call